config/webpack.config.dev.js

```diff
@@ -477,6 +480,15 @@ module.exports = {
         silent: true,
         formatter: typescriptFormatter,
       }),
+      new WriteFilePlugin({
+        test: new RegExp('facetec') 
```

Comment:
1. such pattern is very implicit.
need to use something is more exact, for example

```suggestion
        test: new RegExp('facetec/web/sdk')         
```

2. why this is need ? Doesn't you specified in the `CopyPlugin`'s config what and where it should copy ? Doesn't it enough ?